config.example.toml

```diff
@@ -838,6 +838,16 @@
 # See that option for more info.
 #codegen-backends = rust.codegen-backends (array)
 
+# This is a "runtool" to pass to `compiletest` when executing tests. Tests will
+# execute this tool where the binary-to-test is passed as an argument. Can
+# be useful for situations such as when WebAssembly is being tested and a
+# runtime needs to be configured.
+#
+# This configuration is a space-separated list of arguments so `foo bar` would
+# execute the program `foo` with the first argument as `bar` and the second
+# argument as the test binary.
+#runtime = <none> (string)
```

Comment:
Could you mirror the cargo naming (runner/test runner)? This options seems universally useful, not just for wasm.

Reply:
Excellent suggestion, extracted to https://github.com/rust-lang/rust/pull/122108